dnf-behave-tests/dnf/downgrade.feature

```diff
@@ -52,11 +52,11 @@ Scenario: Downgrade RPM that requires downgrade of dependency
     And package state is
         | package                                | reason     | from_repo     |
         | basesystem-11-6.fc29.noarch            | Dependency | dnf-ci-fedora |
-        | filesystem-3.9-2.fc29.x86_64           | Dependency | dnf-ci-fedora |
+        | filesystem-0:3.9-2.fc29.x86_64           | Dependency | dnf-ci-fedora |
         | glibc-2.28-9.fc29.x86_64               | User       | dnf-ci-fedora |
         | glibc-all-langpacks-2.28-9.fc29.x86_64 | Dependency | dnf-ci-fedora |
         | glibc-common-2.28-9.fc29.x86_64        | Dependency | dnf-ci-fedora |
-        | setup-2.12.1-1.fc29.noarch             | Dependency | dnf-ci-fedora |
+        | setup-0:2.12.1-1.fc29.noarch             | Dependency | dnf-ci-fedora |
```

Comment:
This doesn't pass.
Its odd that the change would affect only a couple of all of the packages.